Scripts/build.ps1

```diff
@@ -6,7 +6,7 @@ param(
 
 $ScriptDir = $PSScriptRoot
 
-Import-Module $ScriptDir\powershell\common.psm1 -Force
+Import-Module $ScriptDir/powershell/common.psm1 -Force
```

Comment:
Good change, but just curious, did this cause some strange issue (as I don't think it has caused any issues before)?

Reply:
no it was just a consistency thing, we use forward slashes elsewhere in our powershell scripts.

Reply:
oh, makes sense, thanks!